<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewCardsDlg</class>
 <widget class="QWidget" name="PreviewCardsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preview cards</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="fact_view_name">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>(fact view name)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="question_label">
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Question:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWebView" name="question" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>295</width>
         <height>100</height>
        </size>
       </property>
       <property name="url" stdset="0">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="answer_label">
         <property name="text">
          <string>Answer:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWebView" name="answer" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>295</width>
           <height>100</height>
          </size>
         </property>
         <property name="url" stdset="0">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="previous_button">
         <property name="text">
          <string>&amp;Previous</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="next_button">
         <property name="text">
          <string>&amp;Next</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="OK_button">
         <property name="text">
          <string>&amp;OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>previous_button</sender>
   <signal>clicked()</signal>
   <receiver>PreviewCardsDlg</receiver>
   <slot>previous</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>next_button</sender>
   <signal>clicked()</signal>
   <receiver>PreviewCardsDlg</receiver>
   <slot>next</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OK_button</sender>
   <signal>clicked()</signal>
   <receiver>PreviewCardsDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
